src/components/Header/NavDropDown.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { MouseEvent } from 'react';
```

Comment:
Let's reuse our Dropdown component instead (and we might need to improve it visually while doing that)